ansible/roles/packages/tasks/install-redhat.yaml

```diff
@@ -1,4 +1,8 @@
 ---
+
+- name: clean yum cache
+  command: yum clean expire-cache
```

Comment:
yum ansible module does not support cleaning cache. Please check Note section from https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html